app_utils/sections/experiment.py

```diff
@@ -1471,6 +1481,121 @@ async def experiment_download_model(q: Q, error: str = ""):
     await q.page.save()
 
 
+async def experiment_quantize_model(q: Q, error: str = ""):
+    experiment = q.client["experiment/display/experiment"]
+    experiment_path = q.client["experiment/display/experiment_path"]
+    logger.info(f"Quantizing model!")
+    zip_path = get_model_path(experiment.name, experiment_path)
+
+    if not os.path.exists(zip_path):
```

Comment:
Check if the model is already quantized:

```suggestion
    if not os.path.exists(quantized_model_dir):
```